Translators/WZDx/geospatial_service.py

```diff
@@ -4,7 +4,21 @@
 import urllib3
 urllib3.disable_warnings()
 from Schemas import geospatial_schemas as geospatial_schemas
+import redis
+import json
 
+cache = None
+
+def get_redis_connection():

```

Comment:
"Get" is usually indicative of the function returning something. Let's rename to `initialize_redis_connecton` or something similar

Reply:
Changed to initialize_redis_connection